README.md

```diff
@@ -69,4 +69,6 @@ Option | Type | Description
 ---|---|---
 `block` | `string[]` | Specify the block names that your testing framework uses. Add a `*` to the end of any string to enable prefix matching (ie. `test*` will match `testExample.only`)<br>Defaults to `["describe", "it", "context", "test", "tape", "fixture", "serial", "Feature", "Scenario", "Given", "And", "When", "Then"]`
 `focus` | `string[]` | Specify the focus scope that your testing framework uses.<br>Defaults to `["only"]`
+`functions` | `string[]` | Specify not permitted functions. Good examples are `fit` or `xit`.<br>Defaults to `[]` (disabled)
+`suffixes` | `string[]` | Specify filename suffixes which contain your test cases to reduce the number of false positives. Rules are only applied to files which match a suffix from the list.<br>Defaults to `[]` (disabled)
```

Comment:
Little bit on the fence about this option, this is something that is already available through ESLint's flat config (or `overrides` in legacy configs). 

Both of these options allow you to specify a separate set of rules that only apply to test files which has the added benefit of avoiding running any of this plugin's code on non-test files.